server/github_client.go

```diff
@@ -71,10 +76,21 @@ type GithubClient struct {
 	Repositories  RepositoriesService
 }
 
-func NewGithubClient(accessToken string) *GithubClient {
+// NewGithubClientWithLimiter returns a new Github client with the provided limit and burst tokens
+// that will be used by the rate limit transport.
+func NewGithubClientWithLimiter(accessToken string, limit rate.Limit, burstTokens int) *GithubClient {
+	const (
+		lruCacheMaxSizeInBytes  = 1000 * 1000 * 1000 // 1Gib
```

Comment:
```suggestion
		lruCacheMaxSizeInBytes  = 1000 * 1000 * 1000 // 1GB
```

https://en.wikipedia.org/wiki/Gibibyte